config/optional/block.block.nodeheaderblock.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - localgov_core
+  theme:
+    - localgov_theme
```

Comment:
Do we need to have a theme dependency for this block?
Is the plan that this block is only for use with the localgov theme or can it be used outside?

Reply:
This doesn't mean it will only work with the locagov theme, but it is needed for the localgov theme. If you were adapting the Croydon theme to work with this you could simple install the theme and place the block.

The dependency is there because this block is automatically enabled and placed when running and site install.

Reply:
Hi Andy, unfortunately, that's how Drupal's block config files work.  You just have to mention a theme.  @see https://git.drupalcode.org/project/drupal/-/blob/8.8.x/core/modules/help_topics/config/optional/block.block.seven_help_search.yml @andybroomfield

Reply:
Following the example of block.block_seven_help_search.yml, this file can be renamed to block.block.localgov_theme_nodeheaderblock.yml.  I cannot suggest anything other than that.

Reply:
Ok, thanks. So it will be possible to place the block, even if localgov_theme is not installed.